ipfx/x_to_nwb/ABFConverter.py

```diff
@@ -423,6 +426,16 @@ def _createAcquiredSeries(self, electrodes):
             for sweep in range(abf.sweepCount):
                 cycle_id = createCycleID([file_index, sweep], total=self.totalSeriesCount)
                 for channel in range(abf.channelCount):
+                    adcName = abf.adcNames[channel]
+
+                    if not self.outputFeedbackChannel:
+                        if adcName in ("IN 0", "IN 1", "IN 2", "IN 3"):
```

Comment:
maybe move the definition of this tuple out of the inner loop and keep it as a constant instance variable?

Reply:
Done!